### PR TITLE
[Merged by Bors] - chore(CategoryTheory/MonoidalCategory): split the monoidal composition

### PR DESCRIPTION
**Pull request by yuma-mizuno**
I realized that the monoid composition `⊗≫` can be defined without depending on the free monoid category, and that it is still useful.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Tactic/CategoryTheory/MonoidalComp.lean [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+to use the whiskering operators `◁` and `▷`, and define the morphism as `V₁ ◁ R V₂ V₃ ▷ V₄ ▷ V₅`.
+However, this morphism has the type `V₁ ⊗ ((V₂ ⊗ V₃) ⊗ V₄) ⊗ V₅ ⟶ V₁ ⊗ ((V₃ ⊗ V₂) ⊗ V₄) ⊗ V₅`,
+which is not what we need. We should insert suitable associators. The desired associators can,
+in principle, be defined by using the primitive three-components associator
+`α_ X Y Z : (X ⊗ Y) ⊗ Z ≅ X ⊗ (Y ⊗ Z)` as a building block, but writing down actual definitions
+are quite tedious, and we usually don't want to see them.
+
+The monoidal composition `⊗≫` is designed to solve such a problem. In this case, we can define
+the desired morphism as `𝟙 _ ⊗≫ V₁ ◁ R V₂ V₃ ▷ V₄ ▷ V₅ ⊗≫ 𝟙 _`, where both `𝟙 _` are
+completed as `𝟙 (V₁ ⊗ V₂ ⊗ V₃ ⊗ V₄ ⊗ V₅)`.
```
Comment (joelriou) [inline Mathlib/Tactic/CategoryTheory/MonoidalComp.lean:29]:
```suggestion
the desired morphism as `𝟙 _ ⊗≫ V₁ ◁ R V₂ V₃ ▷ V₄ ▷ V₅ ⊗≫ 𝟙 _`.
```
The second identity would actually be `𝟙 (V₁ ⊗ V₃ ⊗ V₂ ⊗ V₄ ⊗ V₅)`

**Comment by joelriou** [comment]
Thanks!

bors d+

File: Mathlib/Tactic/CategoryTheory/MonoidalComp.lean (hunk 1 above)
Comment (yuma-mizuno) [reply Mathlib/Tactic/CategoryTheory/MonoidalComp.lean]: Thank you for finding the mistake!

**Pull request closed without merging**